Jupyter/EzPlot_Example/EzPlot_Example.html

```diff
@@ -3,8 +3,6 @@
 title: EzPlot_Example Tutorial
 ---
 
-# EzPlot_Example Tutorial
```

Comment:
We want the download link to the notebook. Why did you remove it?

Reply:
Is the link broken?

Reply:
It wasn't a link. It was just text that didn't do anything.  I fixed this problem for the EzPlot_Example Tutorial in #168 by deleting the useless text and adding in an html link to the notebook.